source/docs/romi-robot/imaging-romi.rst

```diff
@@ -3,44 +3,70 @@ Imaging your Romi
 
 .. warning:: We are working hard on getting the Romi documentation complete. Please check back later.
 
-The Romi has 2 boards that both need to be imaged in order for the robot to work – the Raspberry Pi and the Romi 32U4 Control Board. Once the Raspberry Pi is imaged, we can use the web dashboard included on the device to image the 32U4 Control Board.
+The Romi has 2 microprocessor boards:
+
+#. A **Raspberry Pi** that handles high level communication with the robot program running on the desktop and
```

Comment:
I think a link to an image or product page instead of bold might be more useful?

Reply:
We have hardware links on the hardware page.